### ISSUE TITLE
Proposal: Allow Host / Guest captain to mark / log civs used after each game (and record the result)

### ISSUE THREAD

**Issue by lalitpatel** [SiegeEngineers/aoe2cm2#65]
### Proposal

It would be great if each draft can also record the result of the every matchup and the final score in the draft itself. This would mean that the draft would be act sort of a scorecard for a match-up.

With different marker icons for win and loss, it would be easy to derive the civs selected for each game and record the winner. 

The Host / Guest captain can opt out of this feature while creating a draft. This may be useful for games that will be streamed in the future.

### Things to consider
This may be a spoiler for many viewers who are catching up with games later. To circumvent that we may have users click to expand the section that has the results.

Related to #16

**Comment by HSZemi** [comment]
I see two issues with that mechanic: An organizational one, and a technical one.
- It puts the responsibility to keep scores on the players, one of which would have to mark the used civs accordingly between matches. This could be mitigated by having an admin do that, but that introduces a fourth party into the mix (next to host, guest, spectators) and turns this into another technical issue.
- Currently, a draft is closed and saved after the last turn. Marking civs as used centrally would probably mean that the socket.io connection would need to stay up not just for the few minutes of drafting, but for what are possibly many hours of games, which means the time frame for technical issues like disconnects (and therefore partly marked drafts that get stuck in limbo) gets extended significantly. You would probably also need to tell the draft how many games are being played in order to determine when it can be closed and stored, or add a manual close button for players to click when they added the result of the last game, which you'd have to train them again to not forget…

**Comment by lalitpatel** [comment]
I agree to your arguments. Probably the best way to implement this would be to have user logins (at-least for admins) so users can "own" drafts and then add other meta data to the drafts eg. civ match ups and outcomes. In this case, the socket.io will only be used while drafting and need not stay up for the whole duration of matchup. 

We will need to do the cost benefit analysis for this feature and approach and take a call. Since we preserve the drafts forever, I would say storing the match outcomes would enrich them and they would be able to tell a better story.

**Comment by HSZemi** [comment]
I think Liquipedia does this quite well, and is a more suitable place anyway.

**Comment by HSZemi** [comment]
we have client-side markers by now and Liquipedia as an official record, should be enough I think
